nonprofit-board-management.php

```diff
@@ -5,7 +5,7 @@
 Domain Path: /languages
 Plugin URI: http://wiredimpact.com/nonprofit-plugins/nonprofit-board-management/?utm_source=wordpress_admin&utm_medium=plugins_page&utm_campaign=nonprofit_board_management
```

Comment:
Can we remove the "Plugin URI" since there isn't going to be one moving forward?

Reply:
Great thinking great thinking. I feel like I say that a lot about you lol.